test/token/erc1155/ImmutableERC1155Test.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.20;
+
+import "forge-std/Test.sol";
+import {ImmutableERC1155} from "../../../contracts/token/erc1155/preset/ImmutableERC1155.sol";
+import {IImmutableERC1155Errors} from "../../../contracts/errors/Errors.sol";
+import {OperatorAllowlist} from "../../../contracts/allowlist/OperatorAllowlist.sol";
+import {Sign} from "../../utils/Sign.sol";
+
+contract ImmutableERC1155Test is Test {
```

Comment:
Is testing with forge sufficiently better to have tests written in two different frameworks?

Reply:
Yes forge testing is easier to setup and debug through. Everything is also then written in solidity so we don't have to manage hardhat and typescript libs. I think the in the future we should slowly move the tests over to forge from typescript.